### DISCUSSION TITLE
How to set models folder from Fooocus?

### DISCUSSION THREAD

**Discussion by domrique** [lllyasviel/stable-diffusion-webui-forge#107] [Q&A]
Can i use the same folder with models for Fooocus and Forge?
And set ContolNET models folder from 1111

**Comment by RamonGuthrie** [comment]
> Can i use the same folder with models for Fooocus and Forge? And set ContolNET models folder from 1111

I managed to link my controller models from my Auto1111 
<img width="1174" alt="image" src="https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/156688666/e33fa980-9919-41eb-8b8f-81aa9c631670">
I just need a way to link my Checkpoints and LoRAs

**Reply by rafstahelin** [reply]
In congig.txt command line arcs. It's in the doc. Just look for it

**Comment by ptits** [comment]
> Can i use the same folder with models for Fooocus and Forge? And set ContolNET models folder from 1111

set COMMANDLINE_ARGS= --ckpt-dir "f:\SD_Models"

**Comment by coddoc777** [comment]
there is webui-user.bat
you can uncomment code inside it and set your path to loras and models

**Comment by RamonGuthrie** [comment]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings

**Comment by rafstahelin** [comment]
how do i set controlnet as well? 

set COMMANDLINE_ARGS= --controlnet-dir "path-to-controlnet"
?

Is it possible to do?

**Reply by coddoc777** [reply]
it's already there, integrated

**Reply by rafstahelin** [reply]
No I meant linking the controlnet models to an outside folder 
#Used a symlink. Did not work in commandline args =

**Comment by aib0lit** [comment]
You can just use ntfs links without touching .cmd files. Here is example using Total Commander

https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/14361303/1f79d98b-0109-4122-846f-0bb857ab3f8f
